data_model/src/lib.rs

```diff
@@ -164,10 +165,11 @@ pub enum Parameter {
     Debug,
     Display,
     Clone,
-    PartialEq,
-    Eq,
-    PartialOrd,
-    Ord,
+    IdOrdEqHash,
+    // PartialEq,
+    // Eq,
+    // PartialOrd,
+    // Ord,
```

Comment:
1. commented code
2. we shouldn't need any of these impls on boxed enums

to give you the context, these enums are there just to be able to serialize/encode trait objects. They should be converted into concrete types as soon as deserialized/decoded, otherwise don't use them. Most likely, there is some trait bound that requires these bounds so you won't be able to remove `IdOrdEqHash`